clients/src/main/java/org/apache/kafka/clients/admin/ListTransactionsOptions.java

```diff
@@ -61,6 +62,11 @@ public ListTransactionsOptions filterProducerIds(Collection<Long> producerIdFilt
         return this;
     }
 
+    public ListTransactionsOptions durationFilter(Long durationMs) {
```

Comment:
nit: these can all be primitive longs. (lowercase) There is usage in this file and the Admin file